ais_nginx.conf

```diff
@@ -9,13 +9,26 @@ server {
     # the port your site will be served on
     listen      80;
     # the domain name it will serve for
-    server_name niwen2.aramada.nu; # substitute your machine's IP address or FQDN
+    server_name ais2.armada.nu; # substitute your machine's IP address or FQDN
```

Comment:
server_name {{server_name}};
and then sed in the git/hooks/post-receive is a simple solution